third_party/infineon/cyw30739_sdk/src/wiced_button_manager.c

```diff
@@ -675,6 +676,28 @@ static button_manager_button_t * get_button(platform_button_t id)
     return NULL;
 }
 
+/**
+ * Checks if there is pending event and then re-send the event.
+ *
+ * @return         void : no return value is expected.
```

Comment:
nit: doccommenting returing `void` is a waste of vertical space.
Also this is copied and pasted from the header, so any updates would have to be done twice. It is up to infineon conding standards, so I will merge, however this is the opinion for an external reviewer.